docs/en/using_starrocks/troubleshooting_asynchronous_materialized_views.md

```diff
@@ -331,43 +340,60 @@ To stop a refresh task that occupies too many resources, you can:
   ALTER MATERIALIZED VIEW mv1 INACTIVE;
   ```
 
-- Terminate the running refresh task by using SHOW PROCESSLIST and KILL:
+- Terminate the running refresh task using [CANCEL REFRESH MATERIALIZED VIEW](../sql-reference/sql-statements/data-manipulation/CANCEL_REFRESH_MATERIALIZED_VIEW.md):
 
   ```SQL
-  -- Get the ConnectionId of the running refresh task.
-  SHOW PROCESSLIST;
-  -- Terminate the running refresh task.
-  KILL QUERY <ConnectionId>;
+  CANCEL REFRESH MATERIALIZED VIEW mv1;
   ```
 
-### Materialized view fails to rewrite queries
+### Materialized view query rewrite failure
 
 If your materialized view fails to rewrite relevant queries, you can look into the following aspects:
 
-- **Check whether the  materialized view  and the  query  match.**
+- **Diagnose rewrite failure using TRACE**
+
+  StarRocks supports the TRACE statement for the diagnosis of materialized view rewrite failure:
```

Comment:
StarRocks supports using the TRACE statement to diagnose materialized view rewrite failure: